Make this extension applicable to LogInsights

d-lopes opened the pull request:
make this extension work for AWS CloudWatch LogInsights as well
Adjust Colors (actually differentiate between FontColor and BackgroundColor)

d-lopes committed:
now applicable to LogInsights (+ new colors)

make this extension work for AWS CloudWatch LogInsights as well
Adjust Colors (actually differentiate between FontColor and BackgroundColor)

yusufff commented:
Hey @d-lopes, thank you for the contribution!

I don't have access to an AWS account right now, can you share screenshots of both CloudWatch Logs and LogInsigts so I can make sure the plugin works on both platforms?

d-lopes committed:
optimize display in LogInsights

colorize and format (insert line breaks for @@ tokens) in log details table of LogInsights

d-lopes committed:
screenshots of CloudWatch Logs and Log Insights

d-lopes commented:
sure. i have uploaded them [here](https://github.com/d-lopes/colorize-cloudwatch/tree/patch-1/img) (sorry, i had to anonymise some of the data appearing in the logs). I hope this gives you a good impression. Let me know what you think.

d-lopes committed:
Delete log-group_example.png

d-lopes committed:
Add files via upload

yusufff commented:
LGTM! Thank you!
